src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -1,15 +1,25 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit } from "@angular/core";
 
 @Component({
-  selector: 'app-time-clock',
-  templateUrl: './time-clock.component.html',
-  styleUrls: ['./time-clock.component.css']
+  selector: "app-time-clock",
+  templateUrl: "./time-clock.component.html",
+  styleUrls: ["./time-clock.component.css"]
 })
 export class TimeClockComponent implements OnInit {
+  projects = [
+    { id: "P1", name: "Project 1" },
+    { id: "P2", name: "Project 2" },
+    { id: "P3", name: "Project 3" },
+    { id: "P4", name: "Project 4" }
+  ];
 
-  constructor() { }
+  showFields: boolean;
 
-  ngOnInit(): void {
-  }
+  constructor() {}
+
+  ngOnInit(): void {}
 
+  getShowFields(show: boolean) {
```

Comment:
definitely, the function name should be: `setShowFields`

Reply:
Done 👌